src/index.js

```diff
@@ -187,6 +194,18 @@ exports.createFromB58String = (str) => {
   return new PeerIdWithIs(mh.fromB58String(str))
```

Comment:
Given the fact that B58String is CIDv0, should we replace this with below? 
It would add support for CID, even if someone forgot to replace  `createFromB58String` with `createFromCID`:

```suggestion
  return exports.createFromCID(str)
```

Reply:
This seems reasonable to me